terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionLevelResolverFilter.java

```diff
@@ -32,6 +32,9 @@ protected void doFilterInternal(HttpServletRequest request,
             HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
 
         String path = request.getServletPath();
+        if (path == null || path.length() == 0) {
+            path = request.getPathInfo();
+        }

```

Comment:
- [ ] I got it but please leave comments on the source code for this as following.

// Workaround for application servers that don't provide getServletPath(), eg. WebSphere Liberty Profile 8.5.